openerp/addons/test_impex/tests/test_export.py

```diff
@@ -566,6 +566,9 @@ def test_multiple_records(self):
                 [u'', u'13'],
             ])
 
+    def test_xml_ids(self):
+        self.export(self.commands, fields=['value/id'])
```

Comment:
While this test reproduces the problem, we don't test this module in our CI and other tests fail on our default language.